composites/OnboardingWizard/helpers/ajaxHelper.js

```diff
@@ -87,6 +87,25 @@ var parseHeaders = ( type, config ) => {
 };
 
 /**
+ * @summary Takes the target object and overwrites fields that are undefined
+ * 			or empty with the defaults object values
+ *
+ * @param {Object} target Target to apply default values
+ * @param {Object} defaults Default values
```

Comment:
The params descriptions should be aligned: "Target to apply ..." aligned with "Default values"